lib/src/common/repos_service.dart

```diff
@@ -134,69 +153,89 @@ class RepositoriesService extends Service {
       "has_downloads": hasDownloads,
       "default_branch": "defaultBranch"
     });
-    return _github.postJSON("/repos/${repo.fullName}",
-        body: jsonEncode(data), statusCode: 200);
+    return _github.postJSON(
+      "/repos/${repo.fullName}",
+      body: jsonEncode(data),
+      statusCode: 200,
+    );
   }
 
   /// Deletes a repository.
   ///
   /// Returns true if it was successfully deleted.
   ///
   /// API docs: https://developer.github.com/v3/repos/#delete-a-repository
-  Future<bool> deleteRepository(RepositorySlug slug) {
-    return _github
-        .request('DELETE', '/repos/${slug.fullName}')
-        .then((response) => response.statusCode == StatusCodes.NO_CONTENT);
+  Future<void> deleteRepository(RepositorySlug slug) async {
```

Comment:
I'm ok with this change, but I'm curious what it would be before. Would the resolved bool be null?
The new functionality I assume will either resolve the future successfully or fail/throw?

Reply:
> The new functionality I assume will either resolve the future successfully or fail/throw?

It was already the case before, that's why I made this change. The method would never return false since the `request` method throws when the status code isn't the one expected.

Reply:
ok, that's what I thought. Looks good.